File: tests/testdata/codegen/output-name-conflict-pp/go/output-name-conflict.go
package main

import (
	"github.com/khulnasoft/codeinfra/sdk/v3/go/codeinfra"
	"github.com/khulnasoft/codeinfra/sdk/v3/go/codeinfra/config"
)

func main() {
	codeinfra.Run(func(ctx *codeinfra.Context) error {
		cfg := config.New(ctx, "")
		cidrBlock := "Test config variable"
		if param := cfg.Get("cidrBlock"); param != "" {
			cidrBlock = param
		}
		ctx.Export("cidrBlock", cidrBlock)
		return nil
	})
}

File: tests/testdata/codegen/output-name-conflict-pp/nodejs/output-name-conflict.ts
import * as codeinfra from "@codeinfra/codeinfra";

export = async () => {
    const config = new codeinfra.Config();
    const cidrBlock = config.get("cidrBlock") || "Test config variable";
    return {
        cidrBlock: cidrBlock,
    };
}
